_posts/2016/2016-12-15-hr-ui.md

```diff
@@ -0,0 +1,100 @@
+
+This varies quite a bit depending on the type of task.
+At the moment, for example, we are redesigning our existing software.
+Therefore, we design UI elements, or design and improve new features.
+Sometimes, we also refactor code - so as you see, my work greatly varies from conceptional and design-related work to code-related topics.
+
+I mostly work with LESS, HTML, CSS, GitHub, Source Tree and Photoshop for creating themes, e.g. the home page or category pages.
+At the moment, I don't work a lot with JavaScript, yet I always need to know some basics of ReactJS, whenever I need to change a bit of code in our templates.
+When it comes to our responsive themes, we don't work with Bootstraps, but developed our shop-software themes' mobile responsiveness ourselves.
+We do use Bootstraps though for our MBO pages, for example for the new product detail page, the general settings page and alike.
+If I am developing a new theme, it is often easier to take screenshots, which I then adjust in Adobe Photoshop or Illustrator.
+
+## What does a typical day for an UI-Designer look like?
+
+My day usually starts at 8 o'clock.
+In the morning I am mostly by myself as I am the early bird of my team.
+This way, I have enough time to be concentrated, go through my e-mails, check what tasks are at our board as well as to find out what's new in our intranet Confluence.
+I then get myself a glass of water and start working on my tasks.
+Every now and then I get "interrupted" by a [HipChat](https://www.hipchat.com/) message, which I respond to.
+Then, at 10:30 a.m. we have our daily, where the whole team comes together to talk about current tasks and eventual challenges.
+At 12 o'clock, we have lunch with the team, although I also enjoy to go and have lunch with other teams or bring my own food.
+Depending on whether we have meetings or not, I try to focus on my dedicated tasks in the afternoon.
+
+## What type of tasks are these?
+
+Currently, I'm working on a conceptual task.
+I'm helping out other teams to prepare UI elements, which can be reused and therefore I provide templates, for example, related to payment issues or alike.
+In particular, my colleagues from Jena and I jointly provide a design guideline for Team Ocean since they don't have their own designer.
+Other tasks are designing new features.
+Just yesterday, I designed new icons and I am redesigning our new drag and drop approach for the editor.
+My work is also about bug fixing, code reviews in HTML and LESS.
+
+## What are the required skills for an UI-Designer at ePages?
+
+Regarding soft-skills a UI-Designer should be communicative and shouldn't be afraid to ask colleagues, because we don't sit in only one office, but are spread out.
+I often communicate with my colleagues from Jena or Barcelona.
+Then, considering hard-skills, a UI-Designer needs to know HTML, Less, and should have an understanding of frontend technologies.
```

Comment:
And the hard skills? Well, you should of course know HTML and LESS, and have a good understanding of frontend technologies.